edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -0,0 +1,95 @@
+var edx = edx || {};
+
+(function($) {
+  'use strict';
+
+  var actionToMessageTypesMap = {
+    'submit': {
```

Comment:
See above, I'd love to see these magic strings pulled into a shared constant module